### PR TITLE
feat: Migrate to v3 endpoints pt1

### LINKED ISSUE #690: migrate names related endpoints to v3
### Please describe the problem that your request should resolve.
Use data resources from new version of MDW

### Describe the solution you'd like
 ON HOLD
Bid count missing https://github.com/aeternity/ae_mdw/issues/1916
missing status https://github.com/aeternity/ae_mdw/issues/1954

In the following parts these should be done

TODO IMPROVE DESCRIPTION HERE

- [x] aex141 related (exposed in v3 but not migrated) https://github.com/aeternity/ae_mdw/issues/1898
- [x] aex9 related (exposed in v3 but not migrated)
- [x] contracts/logs
- [x] txs/count (reported bug in Slack)

### Additional context
https://petstore.swagger.io/?url=https://mainnet.aeternity.io/mdw/v3/api

It is expected as a more ongoing process. Will be probably separated into multiple PRs. Collaboration with mdw team will be needed

### PR DESCRIPTION
**Pull request by janmichek**
## Description
related #690

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] change endpoints to v3

### CODE REVIEW

File: src/components/NetworkSelect.vue [hunk 1]
```diff
@@ -16,23 +16,27 @@ import { computed, ref } from 'vue'
 import { useRuntimeConfig } from 'nuxt/app'
 import AppSelect from '@/components/AppSelect'
 
-const config = useRuntimeConfig()
+const {
```
Comment (janmichek) [inline src/components/NetworkSelect.vue:19]: also added related shortening imports of env variables

**Comment by janmichek** [comment]
@Liubov-crypto hint how to perform testing
There should be detailed A/B testing with https://aescan.dev.service.aepps.com/
The app should show precisely same values.

Go to Files Changed tab here in PR
please check changed components and pages. FIles  `/component/*` and  `/pages/*`
please check stores. Files `/store/*`. The whole feature domain should be checked

**Review by Liubov-crypto — APPROVED** [review]
LGTM

**Review by michele-franchi — APPROVED** [review]
Looks good

**Pull request merged**